internal/webhook/v1/aerospikecluster_validating_webhook.go

```diff
@@ -517,6 +555,45 @@ func validateClientCertSpec(cluster *asdbv1.AerospikeCluster) error {
 	return nil
 }
 
+func validateAdminCerts(cluster *asdbv1.AerospikeCluster) error {
```

Comment:
Don't we need a check for TLS somewhere in validation? For PKIOnly auth-mode, the cluster must be TLS enabled